libs/ledger-live-common/src/platform/providers/RemoteLiveAppProvider/api/index.ts

```diff
@@ -20,7 +20,9 @@ export const providers = [
   },
 ];
 
-export function getProviderURL(value: string): RemotePlatformAppProvider {
+export function getProviderURL(
+  value: string
+): RemotePlatformAppProvider["url"] {
```

Comment:
Yes this change was necessary as I was fixing the typings of the `network` method (now using the official parameters of Axios), and therefore `getProviderURL` was returning an object instead of a string, which doesn't work with the `url` param of Axios. 
For `GlobalCatalogProvider` and `RampCatalogProvider` I didn't see any TS error so I didn't have to change anything.